.dockerignore

```diff
@@ -2,12 +2,17 @@
 **
 
 # Exclude folders relevant for build
-!pkg/
+!.git
```

Comment:
It's just a safety net to prevent that anything that is unintended to be included to the Docker image gets added accidentally (e.g., credentials stored locally). Yes, we `COPY` the binaries explicitly, but I'd suggest to keep this file anyways to indeed prevent what I just mentioned. WDYT?

Reply:
Got your point.
What about ignoring the paths that are also in `.gitignore` (like `.kube-secrets`, `dev`, `local` or whatever)? If you store credentials somewhere outside the ignored paths, you would then copy them to the builder image, which normally would not push anyways, I assume...

Reply:
I don't know, I also don't care too much. Both is fine for me. I'd suggest to keep it now as it is as this was also the way it was done before (https://github.com/gardener/gardener/blob/master/.dockerignore), and as the actual goal of the PR is merging g/gext into g/g, so if there is no direct necessity to act here maybe let's postpone it until we hit a problem with it.

Reply:
Sure, sounds good for me.

Reply:
Thanks :)